config.go

```diff
@@ -19,6 +19,7 @@ type serviceConfig struct {
 	kafkaAddr                  *string
 	kafkaHostname              *string
 	kafkaAuth                  *string
+	requestTimeout             *int
```

Comment:
what's the point in general to use pointers for unmutable values like string or ints ?